freqtrade/exchange/exchange.py

```diff
@@ -874,8 +874,15 @@ async def _async_get_candle_history(self, pair: str, timeframe: str,
                 "Fetching pair %s, interval %s, since %s %s...",
                 pair, timeframe, since_ms, s
             )
-
-            data = await self._api_async.fetch_ohlcv(pair, timeframe=timeframe,
+			#fixing support for HitBTC #4778
+            if self.name== 'HitBTC':
+                data = await self._api_async.fetch_ohlcv(pair, timeframe=timeframe,
+                                                        since=since_ms,
+                                                        limit=self.ohlcv_candle_limit(timeframe),
+                                                        params={"sort": "DESC"}
+                                                         )
+            else:
```

Comment:
While i'm sure this will technically work - this is not how this should be done (we'd have 100s of if `exchange=='whatever'` statements ...

Please subclass exchange for hitbtc to fix exchange-specific topics (look at `ftx.py` or `kraken.py` or `bittrex.py` for examples).
Note that you'll also have to add the export to `freqtrade/exchange/__init__.py` to have this subclass used.

From there, i think there's 2 possibilities.
Either, use `_ft_has_default` and `_ft_has` to specify additional parameters for fetch_ohlcv (so this can be configured for other exchanges as well) - defaulting to an empty dict (or `None`) - or alternatively implement `_async_get_candle_history` in the subclass.

Reply:
ok i got it 👌
thanks for hint